src/lib/Key.js

```diff
@@ -149,9 +161,13 @@ class Key {
      * @param {number} validityStartHeight - The validityStartHeight for the transaction
      * @param {Uint8Array | string} [extraData] - Data or utf-8 text to add to the transaction
      * @param {boolean} [isContractCreation]
-     * @returns {Nimiq.ExtendedTransaction} A prepared and signed Transaction object. This still has to be sent to the network.
+     * @returns {Nimiq.ExtendedTransaction} A prepared and signed Transaction object (to be sent to the network)
      */
-    createExtendedTransaction(sender, senderType, recipient, recipientType, value, fee, validityStartHeight, extraData, isContractCreation) {
+    createExtendedTransaction(
+        sender, senderType,
+        recipient, recipientType,
+        value, fee, validityStartHeight, extraData, isContractCreation,
```

Comment:
8-|
Beautiful?

Reply:
Reason: max line length would be more than 120 characters.